shoop/front/views/product.py

```diff
@@ -57,8 +57,7 @@ def get_context_data(self, **kwargs):
         elif product.mode == ProductMode.PACKAGE_PARENT:
             children = (
                 product.get_all_package_children()
-                .language(language)
-                .order_by("translations__name")
+                .translated(get_language())
```

Comment:
get_language is not neccessary here and the .all() shouldn't be either. What happened to the ordering here? Is the .translated even needed without the ordering?